src/web/frontend/components/UserProvider/UserProvider.tsx

```diff
@@ -49,11 +57,17 @@ const UserProvider = ({ children }: any) => {
     axios.post('/api/auth/user', null, { headers: {"Authorization" : `Bearer ${token}`} })
       .then(res => {
         const decoded: JwtPayload = jwt_decode(res.data.jwt_token);
+        const mapped : UserData = {
+          ...decoded,
+          created_at: new Date(),
+          lastonline: new Date(),
+          email:'123@gmail.com',
+};
 
-        if(decoded === defaultUser) {
-          axios.defaults.headers.common["Authorization"] = `Bearer ${token}`;
-          return;
-        }
+      
+        // if(decoded === defaultUser) {
```

Comment:
delete it if not needed

Reply:
it's needed but later after my fixes that I will implement after all merges